x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/Waterfall/accordion_waterfall.tsx

```diff
@@ -41,32 +49,15 @@ const StyledAccordion = euiStyled(EuiAccordion).withConfig({
   .euiAccordion {
     border-top: 1px solid ${({ theme }) => theme.eui.euiColorLightShade};
   }
-  .euiIEFlexWrapFix {
+  .euiAccordion__button {
     width: 100%;
-    height: 48px;
+    height: ${ACCORDION_HEIGHT};
   }
+
   .euiAccordion__childWrapper {
     transition: none;
   }
 
-  .euiAccordion__padding--l {
-    padding-top: 0;
-    padding-bottom: 0;
-  }
-
-  .euiAccordion__iconWrapper {
-    display: flex;
-    position: relative;
-    &:after {
-      content: ${(props) => `'${props.childrenCount}'`};
-      position: absolute;
-      left: 20px;
-      top: -1px;
-      z-index: 1;
-      font-size: ${({ theme }) => theme.eui.euiFontSizeXS};
```

Comment:
Glad to see this gone. Maybe https://github.com/elastic/kibana/issues/101540 can be closed now? Can you try to see how many css selectors a long trace generates now?

Reply:
thanks for linking this issue, I thought about it when I was fixing it but I didn't find it

Reply:
yes, we can close it.